### PR TITLE
Pivotal ID # 186180872: Submission Migrator Task

### PR DESCRIPTION
**Pull request by Juan-EBI**
https://www.pivotaltracker.com/story/show/186180872
- added yaml based itest.
- refactor to use rabbit mq one.

### CODE REVIEW

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -133,6 +133,7 @@ object TestDependencies {
     const val JaxbApi = "javax.xml.bind:jaxb-api:$JaxbApiVersion"
     const val XmlUnitCore = "org.xmlunit:xmlunit-core:$XmlUnitVersion"
     const val XmlUnitMatchers = "org.xmlunit:xmlunit-matchers:$XmlUnitVersion"
+    const val SnakeYaml = "org.yaml:snakeyaml"
```
Comment (jhoanmanuelms) [inline buildSrc/src/main/kotlin/Dependencies.kt:136]: Should we set the specific version?

File: submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/itest/PropertyHolder.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package ac.uk.ebi.biostd.itest.itest
+
+import org.yaml.snakeyaml.DumperOptions
+import org.yaml.snakeyaml.Yaml
+import java.io.File
+
+/**
+ * Holds application properties and write them into its YAML representation.
+ */
+class PropertyHolder {
```
Comment (jhoanmanuelms) [inline submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/itest/PropertyHolder.kt:10]: I'd suggest `ConfigurationPropertiesHolder`, `ConfigurationPropertiesWriter` or `ConfigurationPropertiesHelper` which is a little bit more specific to what it actually does

**Review by jhoanmanuelms — APPROVED** [review]
LGTM. Nice changes! Please check my comments before merging

File: buildSrc/src/main/kotlin/Dependencies.kt (hunk 1 above)
Comment (Juan-EBI) [reply buildSrc/src/main/kotlin/Dependencies.kt:136]: no, it resolved by spring boot so in that way we use the same version as them and there is not risk of imcompability. Actually I was having some issues when using the last version.

**Pull request merged**